Backpack-SwiftUI/Nudger/Classes/BPKNudger.swift

```diff
@@ -106,7 +106,7 @@ public struct BPKNudger: View {
                         if let subtitle {
                             BPKText(subtitle)
                                 .foregroundColor(.textSecondaryColor)
-                                .lineLimit(titleLineLimit())
+                                .lineLimit(nil)
```

Comment:
Can we remove the titleLineLimit() function? I don't think we have any usages left :D

Reply:
Emm, but I find out that we have the title using this above. 🤔

Reply:
The line 104
![image](https://github.com/Skyscanner/backpack-ios/assets/97094570/d7610adc-0ce1-4562-a008-bfbb2c93adc9)
Should we remove too?

Reply:
Fair! We can keep this logic for the title. It will scale to more than 1 line when dynamic type grows larger!